app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
```

Comment:
- [ ] The like model should include a method that updates the like counter for a post, kindly consider implementing this feature

Reply:
Updated